Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs

```diff
@@ -64,7 +68,13 @@ public bool TryGetSwaggerDocument(HttpRequest request, out string document)
 
         private string GetMetadataForContent(OpenApiContent content, HttpRequest request)
         {
-            string basePath = request.PathBase.Value.EnsureSuffixApplied("/") + content.BasePath;
+            var match = _yearSpecificRegex.Match(request.Path);
+
+            var year = match.Success && _isYearSpecific
+                ? "/" + match.Groups[1].Value
+                : string.Empty;
+
+            string basePath = request.PathBase.Value.EnsureSuffixApplied("/") + content.BasePath + year;
```

Comment:
seems like we should be able to reuse the `schoolYearFromRoute` property built in `CreateOpenApiMetadataRequest()` somehow